data/colors/light.scss

```
@@ -286,20 +286,20 @@ $export: (
     warning: $yellow-800,
   ),
 
-  hl: (
-    hover-primary-bg: $blue-500,
+  state: (
+    hover-primary-bg: $blue-500,// `$bg-blue` in Primer CSS
     hover-primary-border: $blue-500,
     hover-secondary-bg: $gray-100,
     hover-secondary-border: $gray-100,
-    selected-primary-bg: $blue-500,
+    selected-primary-bg: $blue-500,// `$bg-blue` in Primer CSS
     selected-primary-border: $blue-500,
   ),
 
   border: (
     primary: $gray-200,
-    primary-light: lighten($gray-200, 3%), // TODO: replace?
-    secondary: $gray-300,
-    tertiary: $gray-000,
+    primary-light: lighten($gray-200, 3%), // TODO: replace? 
+    secondary: $gray-200,
```

With this change, both `border-primary` and `border-secondary` use `gray-200`. Maybe we could do this instead:

```suggestion
    secondary: lighten($gray-200, 3%),
```

Remove `border-primary-light` since it doesn't really make sense to use the word "light" for color modes. And then use `lighten($gray-200, 3%` for `border-secondary`?